src/index.ts

```diff
@@ -383,6 +383,7 @@ const createTab = async ({
     const res = await safeEvaluate(
       loadJS,
       `import(${JSON.stringify(url)})
+        .then(mod => {})
```

Comment:
Here is why this works:

On line 396, that function checks `res` to see if it exists, it assumes that it is an error (that is how it works with runJS). If `res` exists, then it throws it.

In the case of loadJS, `res` is the module object that got imported. So I was seeing a confusing error that said `Thrown: {}`. The addition of this line fixes this bug because it discards the imported module so `res` is undefined. The catch block still works to make `res` exist if something was thrown.